Lib/fontTools/misc/psCharStrings.py

```diff
@@ -251,23 +251,29 @@ class CharStringCompileError(Exception): pass
 
 class SimpleT2Decompiler(object):
 
-	def __init__(self, localSubrs, globalSubrs):
+	def __init__(self, localSubrs, globalSubrs, private):
 		self.localSubrs = localSubrs
 		self.localBias = calcSubrBias(localSubrs)
 		self.globalSubrs = globalSubrs
 		self.globalBias = calcSubrBias(globalSubrs)
+		self.private = private
```

Comment:
So, we pass in private just to lookup isCFF2?  Can't we have isCFF2 passed in directly?